README.md

```diff
@@ -56,7 +56,15 @@ The simplest use case would be:
   ```
 
 `beaver docker build` is just a thin wrapper over `docker build` to add some
-default options (`--pull -t beaver` in particular).
+default options (`--pull -t $img` in particular).
+
+The image name will be taken from the `BEAVER_DOCKER_IMG` environment variable,
+if present. If not present it falls back to the result of `git config
+hub.upstream` (in case you are using the
```

Comment:
But then again, you can always use a different way to get your `BEAVER_DOCKER_IMG` in your script, so I think there is very little gain in getting too smart, I only added `git config hub.upstream` because it was trivial to do so, but I would be more inclined to remove that other than adding more ways to autodiscover.